### PR TITLE
update node SDK to support new paginated roster API

### PR DESCRIPTION
**Pull request by maoyu-dev**
This change adds the helper in node SDK to support the new paginated API to get conversation members.

### CODE REVIEW

File: Node/src/TeamsChatConnector.ts [hunk 1]
```diff
@@ -146,10 +146,45 @@ export class TeamsChatConnector extends builder.ChatConnector {
     });
   }
 
+  /**
+  *  Return members in a team or channel with pagination
+  *  @param {string} serverUrl - Server url is composed of baseUrl and cloud name, remember to find your correct cloud name in session or the function will not find the team.
+  *  @param {string} conversationId - The conversation id or channel id, you can look it up in session object.
+  *  @param {function} callback - This callback returns err or result.
+  *  @param {function} pageSize - How many members to fetch per page
```
Comment (aosolis) [inline Node/src/TeamsChatConnector.ts]: Document the default value if omitted.

[the remark this answers is not in the document]
Comment (maoyu-dev) [reply Node/src/TeamsChatConnector.ts]: Fixed

Comment (maoyu-dev) [reply Node/src/TeamsChatConnector.ts]: Done

File: Node/src/TeamsChatConnector.ts [hunk 2]
```diff
@@ -146,10 +146,45 @@ export class TeamsChatConnector extends builder.ChatConnector {
...
+  *  @param {function} continuationToken - continuationToken if this is a subsequent call to fetch more members
+  */
+  public fetchPagedMembers(
```
[the remark this answers is not in the document]
Comment (maoyu-dev) [reply Node/src/TeamsChatConnector.ts]: Renamed.

File: Node/src/TeamsChatConnector.ts [hunk 3]
```diff
@@ -146,10 +146,45 @@ export class TeamsChatConnector extends builder.ChatConnector {
...
+    serverUrl: string,
+    conversationId: string,
+    callback: (err: Error, result: ChannelAccount[]) => void,
```
[the remark this answers is not in the document]
Comment (maoyu-dev) [reply Node/src/TeamsChatConnector.ts]: The problem is that I want to make pageSize and continuationToken as optional parameters.

File: Node/src/TeamsChatConnector.ts [hunk 4]
```diff
@@ -146,10 +146,77 @@ export class TeamsChatConnector extends builder.ChatConnector {
...
+    let throwInvalidParameter = () => {
+      throw new Error('Invalid parameters: ' + JSON.stringify(args));
+    };
+
+    if (!args) {
+      throwInvalidParameter();
+    }
+
+    switch (args.length) {
+      case 1:
```
Comment (aosolis) [inline Node/src/TeamsChatConnector.ts]: Let's label the case with a comment explaining the expected call pattern, e.g.:
// fetchMembersWithPaging(serverUrl, conversationId, callback)
case 1:
// fetchMembersWithPaging(serverUrl, conversationId, pageSize, callback)
// fetchMembersWithPaging(serverUrl, conversationId, continuationToken, callback)
case 2:

**Pull request merged**